system/jlib/jsocket.cpp

```diff
@@ -1237,6 +1241,9 @@ void CSocket::setKeepAlive(bool set, int time, int intvl, int probes)
 
 void CSocket::checkCfgKeepAlive()
 {
+    if (connectionless())
```

Comment:
harmless, but in the 3 places checkCfgKeepAlive() it is called, it can't be connectionless(),
so only called when known not to be, so perhaps this check here should be removed.

Reply:
very low cost to evaluate, so no harm leaving in.

Reply:
I think udp_create() calls open() calls checkCfgKeepAlive() ?